docs/tikz/arch-multi-standby.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[border=10pt,12pt]{standalone}
+
+\usepackage{pgf}
+\usepackage{tikz}
+\usetikzlibrary{arrows,shapes,snakes}
+\usetikzlibrary{shapes.multipart}
+
+\begin{document}
+
+\begin{tikzpicture}[>=stealth',auto,rounded corners]
+
+  \tikzstyle{app}=[circle,thick,draw=blue!75,fill=white,line width=0.25em]
+  \tikzstyle{primary}=[rectangle split,rectangle split parts=3,
+    text=white,draw=white,fill=blue!75,
+    minimum height=3cm,minimum width=4cm]
+  \tikzstyle{standby}=[rectangle split,rectangle split parts=3,
+    draw=white,fill=blue!35,text=white,
+    minimum height=1.75cm,minimum width=3cm]
+  \tikzstyle{monitor}=[draw=green!35,fill=green!75,
```

Comment:
Well, fact is I am colour blind, and there are different kinds of colour blindness in the wild. I don't use colour schemes or palettes because they are too confusing for me, often resulting in inversion of contrast colours (I failed to be able to use the K9 email client on Android because I would see the already read emails as needing my attention, for example).

So I always do something as basic as possible with colours, and hope that someone who can see them like most other people, or someone who knows what they're doing, would then fix it for me ;-)